### PR TITLE
Use Poison instead of exjsx

### PR DESCRIPTION
**Pull request by davydog187**
Sorry for the notifications for closing multiple PRs.

This PR aims to swap out `exjsx` with `poison` for #115

### CODE REVIEW

**Comment by davydog187** [comment]
I kept the parsing of keys -> atoms. That work will be tracked in #114

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 /doc
 erl_crash.dump
 *.ez
+$TMPDIR/
```
Comment (BlakeWilliams) [inline .gitignore]: I don't think this is needed? Can we remove it?

**Comment by davydog187** [comment]
@BlakeWilliams removed `$TMPDIR` from the gitignore. Not sure why my setup is generating that folder (with BEAM files). I'll figure that out.

**Comment by BlakeWilliams** [comment]
Thanks!

**Pull request merged**